kep/119-node-resource-topology-aware-scheduling/README.md

```diff
@@ -66,7 +66,7 @@ Available resources with topology of the node should be stored in CRD. Format of
 
 ## CRD API
 
-[Code][3] responsible for working with NodeResourceTopology CRD API will be placed in the kubernetes-sigs/scheduler-plugins in pkg/apis/noderesourcetopology-api directory.
+[Code][3] responsible for working with NodeResourceTopology CRD API will imported in the scheduler-plugins repo.
```

Comment:
Let's remove this statement and make initial commit as is.

Reply:
Can you elaborate what you mean. I thought we will be not be placing the API in scheduler-plugins repo in pkg/apis/noderesourcetopology-api directory and be importing it from https://github.com/k8stopologyawareschedwg/noderesourcetopology-api and that's what I have tried to capture here. Am I missing something?

Reply:
right, my fault, I accidentally thought about plugin code too. Right API is imported from  https://github.com/k8stopologyawareschedwg/noderesourcetopology-api